src/components/ModelTableList/CloudGroup.tsx

```diff
@@ -171,10 +171,7 @@ export default function CloudGroup({ filters }: Props) {
     );
   }
   return (
-    <div
-      className="cloud-group u-overflow--auto"
```

Comment:
`u-overflow--auto` wasn't doing anything on these tables as I already removed the horizontal scrolling, so I've removed this class from all tables.